service/src/test/java/uk/nhs/adaptors/gp2gp/common/storage/StorageUtilsTest.java

```diff
@@ -0,0 +1,22 @@
+package uk.nhs.adaptors.gp2gp.common.storage;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.junit.jupiter.api.Test;
+
+public class StorageUtilsTest {
+
+    @Test
+    public void When_FileInputSizeIsNine_Then_ReturnNine() throws IOException {
+        StorageUtils storageUtils = new StorageUtils();
+        String test = "123456789";
+        InputStream is = new ByteArrayInputStream(test.getBytes());
```

Comment:
"is" sounds quite generic. How about "inputStream"?

Reply:
Is this needed? I think we can just use string.length, what do you think